docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -812,6 +853,22 @@ Your new custom cloud is ready to use only after its status changes to
      --google-privilege-bearing-service-account-id "GENERATED_SERVICE_ACCOUNT_ID"
    ```
 
+   Replace the following:
+
+   - `ORGANIZATION_IDENTIFIER` with the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud,
+     for example `org484338bebd2`. You can extract `ORGANIZATION_IDENTIFIER` from the
+     output of the `avn organization list` command.
```

Comment:
These instructions are repeated here quite a few times, would it be possible to instead instruct in one place where to acquire there values, and then have a brief reference to that in others? My concern is that this makes the instructions unnecessarily verbose, and more crucially, makes it possible for the instructions in different locations to drift out of sync later.

Something like "ORGANIZATION_IDENTIFIER - the identifier Aiven organization you obtained in section X / step X" (after first making X exist, of course)?